tsconfig.json

```diff
@@ -42,6 +42,7 @@
     // in other sources:
     "src/index.js",
     "src/standalone.js",
+    "src/languages.js",
```

Comment:
- I think we can remove `src/common/internal-plugins.js` from the list.
- I think `src/languages.js` should move a few lines up to the section with "JavaScript sources affected by JSDoc issues in src/language-*"

Reply:
Will do.